### ISSUE TITLE
I can not certify age.

### ISSUE THREAD

**Issue by iyagicom** [ValveSoftware/steam-for-linux#5377]
#### Your system information

* Steam client version (build number or date):  1516948201
* Distribution (e.g. Ubuntu): ubuntu 17.10 x11
* Opted into Steam client beta?: [Yes/No] y
* Have you checked for system updates?: [Yes/No] y

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large code pastes as a [Github Gist](https://gist.github.com/)

#### Steps for reproducing this issue:

1. I can not change the year and date.
[vokoscreen-2018-02-08_05-05-19.zip](https://github.com/ValveSoftware/steam-for-linux/files/1704595/vokoscreen-2018-02-08_05-05-19.zip)

**Comment by kisak-valve** [comment]
Hello @iyagicom, can you check if the behavior is the same with an X session?

**Comment by iyagicom** [comment]
I'm sorry. It was not wayland that I tested.
I did not know that installing nvidia proprietary drives automatically
changed to x11.
I tested it on x11.
nvidia only runs on x11.

I switched to the x.org server xserver-xorg-nouveau
I tested again with wayland.

Nothing works in wayland.
Steam Launcher, Steam Game does not work at all.
The window does not open.

2018-02-08 6:06 GMT+09:00 kisak-valve <notifications@github.com>:
